src/main/resources/assets/jei/lang/es_es.json

```diff
@@ -109,13 +109,13 @@
   "gui.jei.category.fuel.smeltCount": "Quema %s ítems",
   "gui.jei.category.brewing": "Fermentar",
   "gui.jei.category.brewing.steps": "Pasos: %s",
-  "gui.jei.category.itemInformation": "Informatción",
+  "gui.jei.category.itemInformation": "Información",
 
   "_comment": "Messages",
-  "jei.message.ftblibrary": "Install FTB Library to access ingame config",
+  "jei.message.ftblibrary": "Instala la biblioteca FTB para acceder a la configuración del juego",
```

Comment:
"FTB Library" is the name of the mod, so we should not translate it.

Reply:
Done! Cheers!.